N3.js/src/main/scala/org/w3/banana/n3js/model.scala

```diff
@@ -0,0 +1,7 @@
+package org.w3.banana
+package n3js
+package model
```

Comment:
package model does not match directory - perhaps the file needs renaming and moving to model dir?